pkgs/by-name/up/update-python-libraries/update-python-libraries.py

```diff
@@ -496,7 +496,7 @@ def _update_package(path, target):
         else:
             # forcefully rewrite rev, incase tagging conventions changed for a release
             match = matches[0]
-            text = text.replace(match, f'rev = "refs/tags/{prefix}${{version}}";')
+            text = text.replace(match, f'tag = "{prefix}${{version}}";')
```

Comment:
Seeing this now, we should unquote and uninterpolate the version, when the prefix is empty.